gui/mozregui/wizard.py

```diff
@@ -145,13 +159,16 @@ def nextId(self):
         return ProfilePage.ID
 
 
-class InboundPage(WizardPage):
+class InboundPage(WizardSelectionRangePage):
     UI_CLASS = Ui_Inbound
     TITLE = "Select the inbound changesets range"
     FIELDS = {"start_changeset": "start_changeset",
               "end_changeset": "end_changeset"}
     ID = 2
 
+    def initializePage(self):
```

Comment:
Could you remove this ? This is not required, the base method will be called anyway.